ivy/functional/frontends/__init__.py

```diff
@@ -3,3 +3,90 @@
 from . import jax
 from . import torch
 from . import tensorflow
+import importlib
+
+latest_version = {"torch": "1.12"}
```

Comment:
other frameworks?

Reply:
Yeah, had to add this decided to get your review first.